packages/fluent/scss/tilelayout/_layout.scss

```diff
@@ -0,0 +1,44 @@
+@use "../core" as *;
+@use "_variables.scss" as *;
+
+@mixin kendo-tilelayout--layout() {
+
+    .k-tilelayout {
+        border-width: var( --kendo-tilelayout-border-width, $kendo-tilelayout-border-width );
```

Comment:
the fallback should be with an interpolation: #{$kendo-tilelayout-border-width}

the same goes for all vars